CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+### Recommended changes
+
+#### Change accessible attributes in the markup of the error summary component
```

Comment:
Looking at [other times we've written up recommended changes](https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md#remove-aria-live-from-the-character-count-component), I think we try to be as explicit as possible about what needs to be changed.

Reply:
I've updated this to try to be more descriptive, hopefully it isn't too overwhelming now. Let me know what you think.

Reply:
It's definitely clearer now! Not sure if it's too long, but hopefully Calvin or Claire might be able to help condense it down a little too (or reassure us that it's ok)